python/ray/util/client/worker.py

```diff
@@ -500,14 +514,12 @@ def internal_kv_list(self, prefix: bytes) -> bytes:
     def list_named_actors(self, all_namespaces: bool) -> List[Dict[str, str]]:
         req = ray_client_pb2.ClientListNamedActorsRequest(
             all_namespaces=all_namespaces)
-        return json.loads(
-            self.server.ListNamedActors(req,
-                                        metadata=self.metadata).actors_json)
+        return json.loads(self.data_client.ListNamedActors(req).actors_json)
 
     def is_initialized(self) -> bool:
         if self.server is not None:
             return self.get_cluster_info(
-                ray_client_pb2.ClusterInfoType.IS_INITIALIZED)
+                ray_client_pb2.ClusterInfoType.IS_INITIALIZED) is not None
```

Comment:
I think the return value from the server is a boolean (T/F). I think we want to keep directly returning it.

Reply:
Reverted.